From: Sylvain Le Gall <gildor@debian.org>
Date: Sat, 3 Jul 2010 18:36:39 +0200
Subject: [PATCH] Fix implicit pointer conversion

Avoid segfault on archs where size of pointer is greater than
the size of an integer (patch provided by Dann Frazier).

Author: dann frazier <dannf@hp.com>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=323221
Signed-off-by: Sylvain Le Gall <gildor@debian.org>
---
 runtime/comerror.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/runtime/comerror.c b/runtime/comerror.c
index 1e68542..a137b29 100644
--- a/runtime/comerror.c
+++ b/runtime/comerror.c
@@ -22,6 +22,7 @@
 #include <caml/callback.h>
 #include <caml/fail.h>
 #include <caml/alloc.h>
+#include <caml/printexc.h>
 #include "camlidlruntime.h"
 #include "comstuff.h"
 
-- 
